calico.go

```diff
@@ -178,6 +178,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 			endpoint = api.NewWorkloadEndpoint()
 			endpoint.Metadata.Name = args.IfName
 			endpoint.Metadata.Node = nodename
+			endpoint.Metadata.ActiveInstanceID = args.ContainerID
```

Comment:
But `ActiveInstanceID` is exactly the workload ID, which is already part of the key. It seems strange to store it twice.
The DEL code path should handle already it not being there.